packages/palette/src/_ui-colors.less

```diff
@@ -0,0 +1,34 @@
+@ui-color--black: #061621;
```

Comment:
Want to get your arms dirty with some node scripting?

We can extend the `build` command for this package to include `&& generateStyleSheets.js`

In which `generateStyleSheets.js` is a homegrown node script that will read in `uiColors.ts` (may be a little tricky to get this to work with TS, may be fine to use [ts-node](https://github.com/TypeStrong/ts-node)) and using a plain template output the .less and .sass files for us. 

Additionally when consuming this from LESS I believe someone will have to do: `@import "node_modules/@leafygreen-ui/palette/src/_ui-colors.less`.

I wonder if we should have this generated and put into `styles/ui-colors.less` and have that folder git ignored as well. So the import path looks a little clearer. I'm also fine with `dist/ui-colors.less`. Not sure we need to do the underscore prefix as we're supplying it as a module, but not sure what consumers would prefer.

Reply:
So... I was thinking about that but given the infrequency of changing these, I actually think that it would be less maintenance burden to simply write these separately. That, and transforming the names correctly would be a PITA and more brittle than this.

We could definitely copy the file during the build though.

Reply:
Omitting Less for now, as per our Slack convo.